### ISSUE TITLE
Make math editor work with AT while moving around

### ISSUE THREAD

**Issue by NSoiffer** [benetech/MathEditor#24]
Currently, mathlive has commands to speak the current expression or current selection. Screen readers will echo a character that is typed. What is missing is:
1. Giving feedback when arrowing left/right (also up/down, but not clear on what mathlive means by up/down).
1. Giving feedback for backspace/delete
1. Giving feedback while making a selection (with shift-arrow).
1. Giving feedback after pasting a palette (need to think about what the feedback should be, if any)

Note: this is a feature that needs to be implemented in mathlive.

**Comment by NSoiffer** [comment]
Fixed with update to mathlive.
